### PR TITLE
Feature/more work 20230214

### PR DESCRIPTION
**Pull request by windoverwater**
Previous edits from a few days ago

### CODE REVIEW

**Review by stratofax — DISMISSED** [review]
In general, I think adding numbers and letters to Markdown headers is not needed and adds more maintenance burden to the doc. If you want a version of the doc with a "classic outline" style, you could render it as HTML and attach a CSS that autonumbers headers and subheads.

Also, it's "[Retrospect](https://www.merriam-webster.com/dictionary/retrospect)," not "retro spec" -- which are, I believe, old-timey glasses ("retro specs")

Minor edits, though, nothing to prevent this from dropping onto main.

File: docs/DesignNotes.md [hunk 1]
```diff
@@ -85,24 +85,24 @@ If there is no error, this endpoint returns:
...
 
-## 4) Other Odds and Ends
+## Other Odds and Ends
 
-1. We decided to allow participants in the demo to vote as many times as they would like so to gain experience with RCV and VTP and to allow multiple people to use one phone.  This might be a bad decision - in retro spec it seems like it.  It may be that we want to minimize the number of things that can confuse the voter (even though in real life they will not be voting by phone).  Regardless, to prevent multiple votes the solution would need to work across multiple browsers.
+1. We decided to allow participants in the demo to vote as many times as they would like so to gain experience with RCV and VTP and to allow multiple people to use one phone.  This might be a bad decision - in retrospec it seems like it.  It may be that we want to minimize the number of things that can confuse the voter (even though in real life they will not be voting by phone).  Regardless, to prevent multiple votes the solution would need to work across multiple browsers.
```
Comment (stratofax) [inline docs/DesignNotes.md:105]: So close! "Retrospect" not "retrospec"

Comment (stratofax) [reply docs/DesignNotes.md:105]: Also, you could just delete this whole sentence. It's odd to speculate on what a bad idea a design decision is in this document. Maybe move this to Discussions, or discuss here, if you're not sure about this part of the plan?

**Comment by windoverwater** [comment]
Note - the repo is currently configured to require a new approval with any additional change - which there is one now.  Is that ok?  Whatever you want (as this is more your repo than mine).

FWIIW, I like that setting in my repo but believe it is one of the many owner's choice settings.  But then I also like have the explicit numbers in markdown header lines to help myself and perhaps (?) other ADD people remember where they are.

**Review by stratofax — COMMENTED** [review]
I think if you're not sure about a design decision, we should discuss it and resolve it, instead of questioning it in the doc. This is probably as good a forum as any. Please elaborate on why you think multiple votes from one phone is not good, in retrospect.

File: docs/DesignNotes.md (hunk 1 above)
Comment (windoverwater) [reply docs/DesignNotes.md:105]: I am fine deleting the sentence.  The reason I was thinking it was a bad idea (to be able to vote twice) is that I can see people associating being able to vote multiple times as something inherent with VTP.  Yes we would say that it is not the case, be we are going to be saying a lot of things.  At some point too many things said will be blurred, and the blurring point is different for different folks.  But the tradeoff in this case is ok IMHO - let people vote multiple times.

Comment (windoverwater) [reply docs/DesignNotes.md:105]: Also, I am thinking that changing and discussing the doc is ok as that is what the PR process is for.  However, I am fine if you want to bring the doc back into a discussion topic.  I was just trying it (the process) out as a checked in doc.  You can pick.

**Review by stratofax — APPROVED** [review]
I think this is good to go! 

Also, regarding the many shortcomings of this demo as a "real-world" voting system, like being able to vote multiple times from one device, or allowing dead people to vote, as long as they have a good WiFi connection: we just need to point out that this demo is not secure, and is not intended to be secure, but that was a deliberate decision to make the demo more accessible and encourage people to try it out -- literally one of the main principles of open source software!

Also, I hear the WiFi in heaven is excellent, but in Hell it's provided by AT&T so it's hard to get a good connection...

**Pull request merged**